receiver/zookeeperreceiver/config.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package zookeeperreceiver
+
+import (
+	"time"
+
+	"go.opentelemetry.io/collector/config/configmodels"
+)
+
+type Config struct {
+	configmodels.ReceiverSettings `mapstructure:",squash"`
+
+	// Endpoint to connect to collect metrics. Example: localhost:1234, 10.10.10.10:1234.
+	// Default is ":2181".
+	Endpoint string `mapstructure:"endpoint"`
+
+	// Timeout within which the connection should be established.
+	Timeout time.Duration `mapstructure:"timeout"`
```

Comment:
Is this only for connection or for all the requests?

Reply:
Updated comment to clarify.